test/cmd/lfstest-gitserver.go

```diff
@@ -660,7 +660,7 @@ func gitHandler(w http.ResponseWriter, r *http.Request) {
 		r.Body.Close()
 	}()
 
-	cmd := exec.Command("git", "http-backend")
+	cmd := exec.Command("git-http-backend")
```

Comment:
for some reason that was necessary on my local machine to make the integration tests pass